EventGenerator/src/MuplusProcessGenerator_module.cc

```diff
@@ -0,0 +1,175 @@
+// This module will be used to make electrons from any process originating from mu- stopped in any target material
```

Comment:
This file needs to be removed

Reply:
I agree with Sophie. If this file is no longer needed, we can remove it